djstripe/urls.py

```diff
@@ -26,7 +26,7 @@
 from . import settings as app_settings
 from . import views
 
-
+app_name = 'djstripe'
```

Comment:
Will this affect anyone's current code? i.e. do I have to add it to the backwards-incompatible section of the changelog?

IIRC this makes these urls namespaced if included (It's been a while since I've used ``app_name``)